src/main/java/duke/DukeExceptions.java

```diff
@@ -0,0 +1,38 @@
+package duke;
+
+/**
+ * Custom class of exception for Duke chatbot.
+ */
+public class DukeExceptions extends Exception {
+    protected String printThisOut;
+
+    /**
+     * Constructor to initialize an instance of DukeException
+     * @param commandType String which indicates type of DukeException
```

Comment:
There should be a empty line between description and parameter/return/throws section. This is relevant to all your javadoc comments.